cookiecutter.json

```diff
@@ -1,38 +1,44 @@
 {
-  "project_name": "ClickCLI",
+
+  "project_name": "click-cli-tool",
   "package_name": "{{ cookiecutter.project_name|lower|replace(' ', '_')|replace('-', '_') }}",
   "cli_name": "{{ cookiecutter.package_name }}",
-  "project_version": "0.0.1",
-  "project_description": "This is my click command-line app project.",
+  "project_version": "0.1.0",
+  "project_description": "A CLI tool using Click.",
   "python_version": [
     "3.6",
     "3.7",
-    "3.8"
+    "3.8",
+    "3.9"
   ],
   "virtualenv": [
-    "virtualenv",
-    "python3"
+    "Venv",
+    "Pipenv",
+    "Virtualenv"
   ],
   "linter": [
-    "flake8",
-    "pylint"
+    "Black",
+    "Flake8",
```

Comment:
The `Makefile` uses this value as the value of `LINTER` (the actual executable name).  When it's capitalized, `make lint` can't find application.  I'm not sure what the benefits of capitalizing these options is here, though if it's a valuable change we'll need to make accommodations elsewhere in the templates.  Otherwise I definitely have no problem with lower-case option names.  (Also, the requirements.txt in the template refers to the lower-case names, so we'll need to update those as well if we stick with capitalization.)